src/templates/games/game_form.html

```diff
@@ -4,10 +4,72 @@
   Games
 {% endblock title %}
 {% block content %}
-  <h1>Update or Create Game</h1>
-  <form method="post">
+  <h1>
+    {% if is_create %}
+      Create
+    {% else %}
+      Update
+    {% endif %}
+    Game
+  </h1>
+  <!-- Game Form -->
+  <form method="post" id="game_form">
     {% csrf_token %}
-    {{ form.as_p }}
+    <!-- Manually render each form field -->
+    {% for field in form %}
+      <div>
+        {{ field.label_tag }} {{ field }}
+        {% if field.name == 'name' and is_create %}
+          <!-- BGG Search Button (only under 'name' field in Create view) -->
+          <button type="button" id="bgg_search_button">Search on BoardGameGeek</button>
+          <!-- Container for BGG search results -->
+          <div id="bgg_search_results"></div>
+        {% endif %}
+      </div>
+      <br />
+    {% endfor %}
     <button type="submit">Save</button>
   </form>
+  <!-- AJAX script for BGG search (only in Create view) -->
+  {% if is_create %}
+    <script>
+      document.getElementById('bgg_search_button').addEventListener('click', function() {
+        var search_term = document.getElementById('id_name').value;
+        if (!search_term) return; // Do nothing if search term is empty
+
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
+        fetch(`/games/bgg_search_by_name/?search_term=${encodeURIComponent(search_term)}`)
```

Comment:
To add error handling, we could possibly edit to something like this:

fetch(`/games/bgg_search_by_name/?search_term=${encodeURIComponent(search_term)}`)
    .then(response => {
        if (!response.ok) {
            throw new Error('Network response was not ok');
        }
        return response.json();
    })
    .then(data => {
        // Handle successful response
        // ...
    })
    .catch(error => {
        // Handle fetch or JSON parsing errors
        console.error('Fetch error:', error);
        // Display an error message to the user
    });